content/topics/kubernetes.md

```diff
@@ -6,265 +6,262 @@ url: /kubernetes
+
+detail_sections:
+    - title: Continue using the tools you love
+      description: |
+        Pulumi has first-class support for popular Kubernetes tools, such as Helm, Kustomize,
+        YAML, Secret Managers, Open Policy Agent (OPA) and Custom Resource Definitions (CRDs).
+      cta: Learn More
+      cta_url: "/"
+      items:
+          - title: Everything In One Place
+            description: Easily make the best use of existing Kubernetes tools such as Helm, and reduce the friction caused by multiple deployment tools and models across complex architectures.
+            icon: fa-tools
+
+          - title: Efficient Adoption
+            description: There’s no need to rewrite your existing Kubernetes configurations to get started with Pulumi. You can efficiently adopt existing resources to deploy your application to save time and effort.
+            icon: fa-book
+
+          - title: Secrets Management
+            description: Use Pulumi to ensure secret data is encrypted in transit, at rest, and physically anywhere it gets stored. Bring your own preferred cloud encryption provider or use Pulumi's native secrets provider.
+            icon: fa-key
+
+          - title: Pulumi Kubernetes Operator
+            description: The Pulumi Kubernetes Operator gives you a Kubernetes controller that deploys cloud infrastructure for you and your team.
+            icon: fa-people-carry
+
+    - title: Kubernetes Best Practices with Pulumi Crosswalk
+      description: |
+        Create, deploy, and manage production-ready infrastructure leveraging hosted Kubernetes offerings such as Amazon Elastic Kubernetes Service (EKS), Azure Kubernetes Service (AKS), or Google Kubernetes Engine (GKE).
+      cta: Learn More
+      cta_url: "/docs/guides/crosswalk/kubernetes"
+      items:
+          - title: Day 2 and Beyond
+            description: By using Pulumi Crosswalk for Kubernetes, you can benefit from tried-and-true “Day Two and beyond” integrations and playbooks, improving your infrastructure security, manageability, and cost effectiveness.
+            icon: fa-sun
+
+          - title: Accessible Kubernetes
+            description: Through Pulumi's Crosswalk for Kubernetes library extensions, the authorship experience has improved to make theKubernetes API more accessible and approachable to operators and developers of all backgrounds.
+            icon: fa-users
+
+          - title: Focus on Delivering Value
+            description: With Pulumi you'll focus more on functionality and business logic of your Kubernetes infrastructure and applications, and less on low-level implementation details.
+            icon: fa-chalkboard-teacher
+
+          - title: Avoid Pitfalls
+            description: Discover solutions to the hardest Kubernetes problems to avoid mitigating pitfalls around infrastructure, security, governance, reliability, and maintainability of the cluster, it’s workloads, and underlying resources.
```

Comment:
```suggestion
            description: Discover solutions to the hardest Kubernetes problems to avoid mitigating pitfalls around infrastructure, security, governance, reliability, and maintainability of the cluster, its workloads, and underlying resources.
```